public/main.js

```diff
@@ -1,9 +1,34 @@
 
+// TODO -> this state is one that gets lost every time you refresh the page!
 var state = {
   movieThreads: []
 }
 
+// attempt to get state in the local storage
+var lsState = localStorage.getItem('state');
```

Comment:
`localStorage` is a neat way to deal with persisting state in the browser. Whenever you refresh, the variable `state` is lost. In this nifty way we can keep maintaining it.